src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Definition.cs

```diff
@@ -14,5 +14,37 @@ public static readonly
             RequestType<TextDocumentPosition, Location[]> Type =
             RequestType<TextDocumentPosition, Location[]>.Create("textDocument/definition");
     }
+
+    /// <summary>
+    /// Error object for Definition
+    /// </summary>
+    public class DefinitionError
+    {
+        /// <summary>
+        /// Error message 
+        /// </summary>
+        public string message { get; set; }
+    }
+
+    /// <summary>
+    /// Result object for Definition
+    /// </summary>
+    public class DefinitionResult
```

Comment:
I'd move this out of the Contracts/Definition.cs file - probably create a new file at the LanguageServices folder level. The reasoning is that Contracts files map to data and events sent over the JSON-RPC layer. As this class is used internally and won't be sent on that channel, we shouldn't have it here.